openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -154,6 +154,16 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                         </div>
                     </div>
                 </li>
+                <li id="openassessment_submission_nfile_wrapper" class="field comp-setting-entry">
+                    <div class="wrapper-comp-setting">
+                        <label for="openassessment_submission_nfile_editor" class="setting-label">{% trans "Allow Multiple Files"%}</label>
+                        <select id="openassessment_submission_nfile_editor" class="input setting-input" name="multiple files submission">
+                            <option value="0">{% trans "False"%}</option>
+                            <option value="1" {% if allow_multiple_files %} selected="true" {% endif %}>{% trans "True"%}</option>
+                        </select>
+                    </div>
+                    <p class="setting-help">{% trans "Specify whether learners can upload more than one file. Has no effect if File Uploads Response is None."%}</p>
```

Comment:
If you move this setting up [inside the `openassessment_submission_file_upload_type_wrapper`](https://github.com/edx/edx-ora2/pull/1256/files#diff-4ec864620272da3336bc25826c25b15fR129), it will get hidden along with `File Upload Types` when `File Uploads Response: None`, and shown when `File Uploads Response` is `Required` or `Optional`.